tools/depcheck/pkg/graph/flags.go

```diff
@@ -0,0 +1,317 @@
+package graph
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"path"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/golang/glog"
+
+	"github.com/gonum/graph/concrete"
+)
+
+// GraphOptions contains values necessary to create a dependency graph.
+type GraphOptions struct {
+	// RepoImportPath is the "go-import path" of the repository being traversed
+	RepoImportPath string
```

Comment:
What is this used for?

Reply:
> What is this used for?

Wait, for output?  No.  Remove it from options and instead collect it in the TraceFlags.ToOptions method.  The display name is associated with Trace, not the graph.  At least the way its currently written